aws-lambda-api-gateway-proxy/lambda.tf

```diff
@@ -0,0 +1,38 @@
+# Lambda
```

Comment:
None of the other files have comments. For consistency, should we remove this and the one below?